apps/klasses/templates/klasses/klass_form.html

```diff
@@ -5,46 +5,57 @@
 {% block content %}
     <div class="ui vertical masthead container">
         <div class="ui center aligned grid">
-            <div class="ui fourteen wide column">
+            <div class="ui ten wide column">
 
                 <form class="ui form" method="post" enctype="multipart/form-data">
                     {% csrf_token %}
 
                     {{ form.non_field_errors }}
+                    <div class="two fields">
 
-                    <div class="field">
-                        {{ form.title.errors }}
-                        {{ form.title.label_tag }}
-                        {{ form.title }}
-                    </div>
+                        <div class="eleven wide field">
+                            {{ form.title.errors }}
+                            {{ form.title.label_tag }}
+                            {{ form.title }}
+                        </div>
 
-                    <div class="field">
-                        {{ form.course_number.errors }}
-                        {{ form.course_number.label_tag }}
-                        {{ form.course_number }}
-                    </div>
+                        <div class="five wide field">
+                            {{ form.course_number.errors }}
+                            {{ form.course_number.label_tag }}
+                            {{ form.course_number }}
+                        </div>
 
-                    <div class="field">
-                        {{ form.description.errors }}
-                        {{ form.description.label_tag }}
-                        {{ form.description }}
                     </div>
 
-                    <div class="field">
-                        {{ form.image.errors }}
-                        {{ form.image.label_tag }}
-                        {{ form.image }}
-                    </div>
+                        <div class="sixteen wide text field">
+                            {{ form.description.errors }}
+                            {{ form.description.label_tag }}
+                            {{ form.description }}
+                        </div>
```

Comment:
Move this field outside the `two fields` div.

Reply:
I misread this. It's not inside the `two fields` div. It just looks like it, because it's indented too far.

Reply:
Yep I see the Issue. Thanks